lib/elixir/lib/function.ex

```diff
@@ -2,10 +2,10 @@ defmodule Function do
   @moduledoc """
   A set of functions for working with functions.
 
-  We can have two types of captured functions: external and local.
+  There are two types of captured functions: **external** and **local**.
   External functions are functions residing in modules that are captured
-  with `&/1`, such as `&String.length/1`. Local are the anonymous functions
-  defined with `fn/1` or with the capture operator `&/1` using `&1`, `&2`
+  with `&/1`, such as `&String.length/1`. Local functions are anonymous functions
+  defined with `fn/1` or with the capture operator `&/1` using `&1`, `&2`,
   and so on as replacements.
```

Comment:
Would it make sense to add an example using `Function.info(fn -> nil end, :type) => :local` and `Function.info(&String.length/1, :type) => :external`?

Reply:
We have that example in `Function.info/1`, I think it's fine to skip it here.